core_perception/lidar_localizer/nodes/ndt_matching/ndt_matching.cpp

```diff
@@ -130,8 +130,9 @@ static pcl_omp::NormalDistributionsTransform<pcl::PointXYZ, pcl::PointXYZ> omp_n
 static int max_iter = 30;        // Maximum iterations
 static float ndt_res = 1.0;      // Resolution
 static double step_size = 0.1;   // Step size
-static double trans_eps = 0.01;  // Transformation epsilon
-
+static double trans_eps = 0.001;  // Transformation epsilon. In PCLv1.10 (ros noetic) this value is squared error not base epsilon 
+                                  // NOTE: A value of 0.0001 can work as well. 
+                                  // This will increase the required iteration count (and therefore execution time) but might increase performance 
```

Comment:
Just to clarify, the mentioned performance here is in terms of accuracy? So the tradeoff is speed vs accuracy?

Reply:
Yeah, I'll clarify